Assets/Modules/Varietas Delectat/XML/UnitArtStyles&CivInfos/VD_Egypt_CIV4UnitArtStyleTypeInfos.xml

```diff
@@ -107,36 +107,36 @@
 					<UnitType>UNIT_SPEARMAN</UnitType>
 					<UnitMeshGroup>
 						<EarlyArtDefineTag>ART_DEF_UNIT_SPEARMAN_EGYPT</EarlyArtDefineTag>
-						<MiddleArtDefineTag>ART_DEF_UNIT_SPEARMAN_EGYPT</MiddleArtDefineTag>
```

Comment:
In this file you replace all Egyptian unit art with art from SoI but surely at least for the ancient and classical eras the traditional art should be reserved so that it still represents Pharaonic Egypt?

Reply:
Yes! At least, for the Ancient and Classical Era units. This is the best workaround to show the accurate flavor imo, similar to the city art already in place.

Reply:
But that's not the case with the unit art right now, is it? Stuff like Ayyubid spearmen suggests Middle Eastern art.

Reply:
Sorry for the late reply, I did not see this. The Middle Eastern art is used for the <MiddleArtDefineTag> and the <LateArtDefineTag> tags: the former is used in the Medieval and Renaissance Eras, while the latter is used from the Industrial Era onwards. I left the <EarlyArtDefineTag> untouched, because that defines the Pharaonic Egyptian art in the Ancient and Classical Eras.

Reply:
Ah, didn't notice that. In that case it works well.